hrui/hrui.php

```diff
@@ -29,6 +29,7 @@
 
 function hrui_civicrm_pageRun($page) {
   CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.hrui', 'css/hrui.css');
+  CRM_Core_Resources::singleton()->addStyleFile('org.civicrm.hrui', 'css/contact.css');
 
```

Comment:
Sorry for mentioning this late but I just found that it will be more convenient to move the whole code inside this if statement :

`if($context == 'page' && $tplName == "CRM/Contact/Page/View/Summary.tpl" ) {`

in hrui_civicrm_alterContent hook , to a new function at the end of the same file ( hrui.php ) ...

please create a method called for example _hrui_updateContactSummaryUI() which return the content that going to be added to the page , so what inside the beforementioned if statement will change to something like this :

```php
  // fetch data to the new summary page UI
  if($context == 'page' && $tplName == "CRM/Contact/Page/View/Summary.tpl" ) {
       $content = _hrui_updateContactSummaryUI();
   }
```